bootstrap_actions/resume_step.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+source /opt/emr/logging.sh
+
+STEP_TO_START_FROM_FILE=/opt/emr/step_to_start_from.txt
+
+function resume_from_step() {
+    if [[ -f $STEP_TO_START_FROM_FILE ]]; then
+        log_pdm_message "Previous step file found"
+        STEP=`cat $STEP_TO_START_FROM_FILE`
+        CURRENT_FILE_NAME=`basename "$0"`
+        FILE_NAME_NO_EXT="${CURRENT_FILE_NAME%.*}"
```

Comment:
```suggestion
        FILE_NAME_NO_EXT="$${CURRENT_FILE_NAME%.*}"
```

this might want to be double dollar signs. But I'm sure we applied it and it worked...